Realm/RLMProperty.h

```diff
@@ -92,4 +97,17 @@ RLM_ASSUME_NONNULL_BEGIN
 
 @end
 
+
+/**
+ This class describes a specific property on a given class.
+ */
+@interface RLMPropertyDescriptor : NSObject
+
++ (instancetype)descriptorWithClass:(Class)objectClass propertyName:(NSString *)propertyName;
```

Comment:
Looks like that is [SE-0062](https://github.com/apple/swift-evolution/blob/master/proposals/0062-objc-keypaths.md). It'd be nicer if it was more than a compiler-checked string literal since using that will mean restating the type in both arguments to the `LinkingObjects` initializer.